contracts/FastBreakV1.cdc

```diff
@@ -624,25 +638,25 @@ pub contract FastBreakV1: NonFungibleToken {
 
     /// Capabilities of Fast Break Players
     ///
-    pub resource interface FastBreakPlayer {
-        pub fun play(
+    access(all) resource interface FastBreakPlayer {
+        access(Play) fun play(
             fastBreakGameID: String,
             topShots: [UInt64]
         ): @FastBreakV1.NFT
     }
 
     /// Fast Break game collection
     ///
-    pub resource Collection:
+    access(all) resource Collection:
         NonFungibleToken.Provider,
         NonFungibleToken.Receiver,
-        NonFungibleToken.CollectionPublic,
+        NonFungibleToken.Collection,
         FastBreakNFTCollectionPublic
     {
 
-        pub var ownedNFTs: @{UInt64: NonFungibleToken.NFT}
+        access(all) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}
```

Comment:
Make sure that this is `access(self)` so it can't be accessed by casting a public reference

Reply:
We'll actually need to make this `access(all)` based on a [new FLIP](https://github.com/onflow/flips/pull/263/files) and [changes to the standard](https://github.com/onflow/flow-nft/pull/126/commits/68c66a2d5a890acd480175ea9f63d9d10c3d0d38)